app/src/ui/changes/filter-changes-list.tsx

```diff
@@ -1142,6 +1149,10 @@ export class FilterChangesList extends React.Component<
     const disableAllCheckbox =
       files.length === 0 || isCommitting || rebaseConflictState !== null
 
+    const toBeCommittedFilterText = this.state.filterToIncludedCommit
+      ? 'Remove to be committed filter'
```

Comment:
Sounds a bit odd maybe? 🤔  What about… ?
```suggestion
      ? 'Show all changed files'
```

Reply:
Yeah. I know..  But "Show all changed files" is inaccurate. A user could have a typed in filter so it it would be "Show all "text" filtered changes".  Thus.. the struggle for a concise description.

Reply:
Other ideas I had:
"Show checked and unchecked files" with inverse being "Show only checked files" 
"Show included and unincluded files" with inverse being "Show only included files"
"Remove filter" (not as direct.. but maybe clear enough?) 

Other thoughts:
- Making the "Remove" action say "Show" feels like it is applying a filter when it is not?
- Do we even like the button design? Another design I would like to explore/discuss (this PR was just for MVP) would be that the filter be prepended with a "Filter dropdown" that has a checkbox list of filters you can apply.

Reply:
Like: 
![Showing repository list UI](https://github.com/user-attachments/assets/929fbf32-d03e-4da8-9217-41f1454b152e) 
Mixed with:
![Showing file changes list filter on github](https://github.com/user-attachments/assets/e2d6d728-9ddc-4561-9b00-07363da6e428)

Reply:
I'm not fond of the button design, no, but I think that's something we can iterate on!

I don't think `Show` means filtering, it just talks about the actual consequences of the action, what the user will see. `Show included and non-included files` would be the least bad option IMO 😂  I can't think of a perfect wording for this 😓  (I replaced `unincluded` with `non-included` because the former sounds weird to me, but I'm a non-native English speaker!).

Reply:
You are right "unincluded" is not a word. :D

Reply:
Changed to:  (Pretty verbose, but hopefully easy to understand and suitable for MVP)

'Show files included and not included in the commit'
'Only show files to be included in the commit'

![CleanShot 2025-01-17 at 09 01 02](https://github.com/user-attachments/assets/4c6c36a2-0d65-4935-ae64-3f45c90f6b3c)